.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDatePeriodParserConfiguration.cs

```diff
@@ -804,6 +872,18 @@ private DateTimeResolutionResult ParseYear(string text, DateObject referenceDate
                 var endDay = DateObject.MinValue.SafeCreateFromValue(year + 1, 1, 1);
 
                 ret.Timex = year.ToString("D4");
+
+                if (halfType == 1)

```

Comment:
can we avoid this magic number?

Reply:
If such numbers are really needed, please declare as Constants.